src/OmniSharp.Http/Startup.cs

```diff
@@ -45,8 +52,15 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                                 !category.Equals(projectEventForwarder, StringComparison.OrdinalIgnoreCase)));
                 });
 
+            var options = serviceProvider.GetRequiredService<IOptionsMonitor<OmniSharpOptions>>();
+            var plugins = new PluginAssemblies(_commandLinePlugins.Concat(options.CurrentValue.Plugins.GetNormalizedLocationPaths(_environment)));
```

Comment:
This line is too long

Reply:
I would like to push back in this one. The line's length AFAICS is comparable or even shorter many other lines in the same file and in the repo and breaking in up IMO will hurt readability.

---
In reply to: [341832182](https://github.com/OmniSharp/omnisharp-roslyn/pull/1615#discussion_r341832182) [](ancestors = 341832182)